sys/android-architecture.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if [ "$#" -ne 1 ]; then
+	echo "Please enter an android architecture..."
+fi
```

Comment:
Even if $# is not equal to 1, the default case will be done and there it is an `exit 1`